datasets/statistieken/dataset.json

```
@@ -6,60 +6,279 @@
   "description": "Gepubliceerde statistieken conform generiek model",
   "version": "0.0.1",
   "owner": "Gemeente Amsterdam",
-  "publisher": "Datateam Basisstatistiek",
+  "publisher": "Onderzoek en Statistiek",
   "crs": "EPSG:28992",
-  "creator": "Team Basisstatistiek",
+  "creator": "Onderzoek en Statistiek",
   "auth": "OPENBAAR",
   "authorizationGrantor": "basisstatistiek.ois@amsterdam.nl",
   "tables": [
     {
-      "id": "measures",
-      "$ref": "measures/v1.0.0",
-      "activeVersions": {
-        "1.0.0": "measures/v1.0.0"
+      "id": "indicatoren",
+      "type": "table",
+      "version": "1.0.0",
+      "description": "Alle indicatoren (variabelen) en hun metadata",
+      "schema": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "type": "object",
+        "additionalProperties": false,
+        "identifier": [
+          "id"
+        ],
+        "required": [
+          "schema",
+          "id",
+          "naam",
+          "label",
+          "definitie",
+          "thema"
+        ],
+        "display": "naam",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "type": "integer",
+            "title": "id",
+            "description": "Unieke identificatie van de indicator"
+          },
+          "naam": {
+            "type": "string",
+            "title": "naam",
+            "description": "Unieke naam van de indicator"
+          },
+          "label": {
+            "type": "string",
+            "title": "label",
+            "description": "Beknopte label van de indicator t.b.v. weergave"
+          },
+          "label_uk": {
```

Underscores zijn hier niet toegestaan.

```suggestion
          "labelUk": {
```